File: JAVA/factorial.java
import java.util.Scanner;
class factorial {
	public static void main (String [] args) {
		Scanner SC = new Scanner (System.in);
		int x;
		int results;
		System.out.println("User, enter a number: ");
		x = SC.nextInt();
		results = factorial(x);
		System.out.println(x + " --> " + results);
	}
	public static int factorial (int num) {
		int res = 1;
		while (num > 0) {
			res *= num;
			num -= 1;
		}
		return res;
	}
}

File: C++/factorial.cpp
# include <iostream>
using namespace std;
int main () {
	int x;
	int orig;
	int res = 1;
	cout << "User, enter a number: ";
	cin >> x;
	orig = x;
	while (x > 0) {
		res *= x;
		x--;
	}
	cout << orig << " --> " << res << "\n";
}
